### PR TITLE
CTM-ETM coupling 5b deliverable 2 & 3 

### PR DESCRIPTION
**Pull request by kndehaan**
This PR contains updates of user output for deliverable 2&3 of the CTM-ETM coupling projects. The focus is to update outputs considering changes in modelling of:

- methanol
- waste mix
- oil mix
- ammonia

The (queries for the) following elements are updated:

- Data-exports
- Charts
- Dashboard items
- MYC

Goes with:
- https://github.com/quintel/etsource/pull/3093
- https://github.com/quintel/etmodel/pull/4302
- https://github.com/quintel/multi-year-charts/pull/49

### CODE REVIEW

File: config/interface/output_element_series/use_of_final_demand_in_industry_other_metals.yml [hunk 1]
```diff
@@ -47,15 +59,15 @@
   dependent_on: 
   output_element_key: use_of_final_demand_in_industry_other_metals
   key: electricity_use_of_final_demand_in_industry_other_metals
-- label: heat
-  color: "#8B0000"
```
Comment (mabijkerk) [inline config/interface/output_element_series/use_of_final_demand_in_industry_other_metals.yml:51]: Why is heat removed here? There is a `industry_final_demand_for_metal_other_metals_steam_hot_water` node in the other metals sector.

File: config/interface/output_element_series/industrial_heat_mekko.yml [hunk 2]
```diff
@@ -120,13 +120,73 @@
...
+  order_by: 301
+  group: supply
+  show_at_first: 
+  is_target_line: false
+  target_line_position: 
+  gquery: industry_heat_backup_burner_coal_in_industrial_heat_network_mekko
+  is_1990: 
+  dependent_on: 
+  output_element_key: industrial_heat_mekko
+  key: industry_heat_backup_burner_coal_in_industrial_heat_network_mekko_supply_industrial_heat_mekko
```
Comment (mabijkerk) [inline config/interface/output_element_series/industrial_heat_mekko.yml:134]: @kndehaan I think we can revert the changes in this file right?

File: config/locales/interface/output_element_series/nl_labels.yml [hunk 3]
```diff
@@ -1128,7 +1133,7 @@ nl:
         nationally_refined: "Raffinaderij"
         natural_gas: "Aardgas"
         natural_gas_national_transport: "Aardgas voor nationaal transport"
-        natural_gas_and_derivatives: "Netwerkgas (aard-/groengas)"
```
Comment (mabijkerk) [inline config/locales/interface/output_element_series/nl_labels.yml:1131]: Why was this change made?

File: config/interface/output_element_series/industrial_heat_mekko.yml (hunk 2 above)
Comment (kndehaan) [reply config/interface/output_element_series/industrial_heat_mekko.yml:134]: Yes, all changes are related to the backup burner.

**Comment by mabijkerk** [comment]
The inputs and output of waste in the Sankey chart do not seem to add up:
image

**Comment by kndehaan** [comment]
> The inputs and output of waste in the Sankey chart do not seem to add up:

This is now fixed
